bedrock/externalpages/templates/externalpages/pocket/platform-pages.html

```diff
@@ -0,0 +1,31 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% extends "externalpages/pocket/base.html" %}
+
+{% from "macros-protocol.html" import split with context %}
+
+{% block page_title %} Add Pocket to Chrome {% endblock %}
+
+{% block pocket_css %}
+ {{ css_bundle('pocket-platforms') }}
+{% endblock %}
+
+{% block content %}
+  {% include 'externalpages/pocket/includes/platform-nav.html' %}
+    <div class="platform">
+        {% call split(
+          block_class='mzp-l-split-center-on-sm-md mzp-t-content-lg mzp-l-split-reversed mzp-l-split-hide-media-on-sm-md',
+          media_class='mzp-l-split-h-center',
+          image_url=platform_image,
+          )%}
+          <h1>{{ self.page_title() }}</h1>
+          <p>{{ self.page_desc() }}</p>
+          <a href="{{ self.page_url() }}" class="mzp-c-button" target="_blank" rel="external noopener">{{ self.page_cta() }}</a>
```

Comment:
This template gives me an error `jinja2.runtime.TemplateReference object has no attribute page_url`